.github/workflows/tests.yml

```diff
@@ -38,16 +38,48 @@ jobs:
           rustup target install aarch64-apple-ios
       - name: Tests
         run: cargo test --verbose
+      - name: Build w/ Evercrypt
+        if: matrix.os != 'macos-latest'
```

Comment:
We are we leaving out chips for evercrypt? Isn't this working now?

Reply:
Evercrypt doesn't support AES-GCM on old chips. And GH actions mac machines are too old.